### PR TITLE
Add payment icons right next to the Discover other payments link

### LINKED ISSUE #33626: Payments Settings: When WCPay is installed and setup the current version of the page is wrong
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When WCPay is installed and setup, we shouldn't display the `Recommended ways to get paid` box in the payment settings as it presents competitive alternatives to WCPay when it's already setup. Instead we should direct users to the marketplace where they can see the payment options that will enhance their store. 

### Expected behavior

[As in this Figma design](e428PXC0QyEwYqtX48lLq0-fi-12830%3A96246): 

e428PXC0QyEwYqtX48lLq0-fi-12830%3A96246

When WCPay is installed and setup:

- [ ] We should not display the recommended additional payment providers at the bottom of the Payment Methods page. 
- [ ] The link should say: `Discover additional payment options`
- [ ] Next to the link we should include selected payment options logos
- [x] The link should direct the user to the external Marketplace with BNPL filter https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/wallets/?categoryIds=28681&collections=product&page=1

### Actual behavior

Currently this page includes a box with the recommended payment providers
The link goes to the wrong place
Screenshot 2022-06-27 at 15 45 42

### Steps to reproduce

1. Onboard to WooCommerce from a WCPay eligible country 
2. Install WCPay
3. Set up WCPay
4. Go to Payments Settings

### WordPress Environment

WC Version: 6.6.1
REST API Version: ✔ 6.6.1
WC Blocks Version: ✔ 7.6.2
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.0

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by moon0326**
### All Submissions:

-   [X] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [X] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Partially closes #33626 .

This PR adds payment icons right next to the Discover other payments link.

### How to test the changes in this Pull Request:

Case 1: WC Pay is not installed and setup

1. Start with a fresh install.
2. Complete OBW with a WC Pay eligible country.
3. Navigate to `WooCommere -> Settings -> Payments`
4. You should see Stripe and Paypal icons when country is the US

Case 2: WC Pay is setup

1. Start with a fresh install
2. Complete OBW with a WC Pay eligible country.
3. Set up WCPay OR open `./includes/class-wc-payment-gateway-wcpay.php` from WooCommerce Payments plugin. Return `false` from the `needs_setup()` method. This will make WooCommerce think that WC Pay is already set up.
3. Navigate to `WooCommere -> Settings -> Payments`
4. You should see Klarna, Paypal, Afterpay, Amazon Pay, and Affirm icons when country is the US

### Screenshots

Screen Shot 2022-07-05 at 8 29 37 PM

### Other information:

-   [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by chihsuan** [comment]
Hey @moon0326 Do we still need [plugins/woocommerce/assets/images/payment_methods/72x72/stripe2.png](https://github.com/woocommerce/woocommerce/pull/33744/files#diff-bb91a65f23a225887832622f6845c843c74fee47feca3d751fa3146418508346)?

And I think we should add back the "check plugin state" logic? https://github.com/woocommerce/woocommerce/pull/33744/commits/4643c0a7b42379884cb9d319ac127b8322d434bf#diff-6b251c63d49fc831276bd7fcf9cf3c6ee8d1044f0a1c021a1866dad1daeb7b83L252-L254.

**Comment by moon0326** [comment]
> Hey @moon0326 Do we still need [plugins/woocommerce/assets/images/payment_methods/72x72/stripe2.png](https://github.com/woocommerce/woocommerce/pull/33744/files#diff-bb91a65f23a225887832622f6845c843c74fee47feca3d751fa3146418508346)?
> 
> And I think we should add back the "check plugin state" logic? [4643c0a#diff-6b251c63d49fc831276bd7fcf9cf3c6ee8d1044f0a1c021a1866dad1daeb7b83L252-L254](https://github.com/woocommerce/woocommerce/commit/4643c0a7b42379884cb9d319ac127b8322d434bf#diff-6b251c63d49fc831276bd7fcf9cf3c6ee8d1044f0a1c021a1866dad1daeb7b83L252-L254).

Good catch! We don't need it anymore.

I'll add back the active plugins logic.

**Comment by moon0326** [comment]
> Hey @moon0326 Do we still need [plugins/woocommerce/assets/images/payment_methods/72x72/stripe2.png](https://github.com/woocommerce/woocommerce/pull/33744/files#diff-bb91a65f23a225887832622f6845c843c74fee47feca3d751fa3146418508346)?
> 
> And I think we should add back the "check plugin state" logic? [4643c0a#diff-6b251c63d49fc831276bd7fcf9cf3c6ee8d1044f0a1c021a1866dad1daeb7b83L252-L254](https://github.com/woocommerce/woocommerce/commit/4643c0a7b42379884cb9d319ac127b8322d434bf#diff-6b251c63d49fc831276bd7fcf9cf3c6ee8d1044f0a1c021a1866dad1daeb7b83L252-L254).

Removed the image in https://github.com/woocommerce/woocommerce/pull/33744/commits/ed37449f0be142122868c5c346124fee575bc79a and added back the active plugin logic in https://github.com/woocommerce/woocommerce/pull/33744/commits/59d524f3ab9b624ddf1bfc44a96cde083bd8c11d

File: plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways.php [hunk 1]
```diff
@@ -225,8 +228,31 @@ class_exists( 'WC_Payments_Account' )
...
+								$filter_by = 'category_additional';
+							} else {
+								$link_text = __( 'Discover other payment providers', 'woocommerce' );
+								$filter_by = 'category_other';
+							}
+
+							$plugin_suggestions = array_filter(
+								$plugin_suggestions,
+								function( $plugin ) use ( $country, $filter_by, $active_plugins ) {
+									if ( ! isset( $plugin->{$filter_by} ) || ! isset( $plugin->image_72x72 ) || in_array( $plugin->id, $active_plugins, true ) ) {
```
Comment (chihsuan) [inline plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways.php]: I installed the WooCommerce Stripe Payment Gateway, but I can still see the Stripe icon. 🤔 

I think we should check the `plugins` property instead.

https://github.com/woocommerce/woocommerce/blob/59d524f3ab9b624ddf1bfc44a96cde083bd8c11d/plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php#L44

**Review by chihsuan — CHANGES_REQUESTED** [review]

File: plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways.php (hunk 1 above)
Comment (moon0326) [reply plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways.php]: Good catch 👍 

Updated in https://github.com/woocommerce/woocommerce/pull/33744/commits/4d8dbd85cb8000edd0178adfc82f012ed038be38

It assumes the first array item is the plugin ID

**Review by chihsuan — APPROVED** [review]
Thanks @moon0326. Looks good and tested well. 👍

**Pull request merged**